media_player/alexa.py

```diff
@@ -250,7 +272,7 @@ def refresh(self, device):
                 self._media_title = self._session['infoText']['title']
                 self._media_artist = self._session['infoText']['subText1']
                 self._media_album_name = self._session['infoText']['subText2']
-                self._media_image_url = self._session['mainArt']['url']
+                self._media_image_url = self._session['mainArt']['url'] if 'url' in self._session['mainArt'] else None
```

Comment:
I'd prefer to keep maximum line length to 80 characters. (I think this is the requirement for official components for HA.)

Reply:
It is. I was being lazy to try to figure out if it resolved the Decode bug. A lot of the Amazon API access needs to be moved out to pypi anyway before this can be submitted so I wasn't running it though the HA linter.